src/test/java/io/dropwizard/cassandra/test/DropwizardCassandraIT.java

```diff
@@ -75,8 +75,7 @@ public void canQueryCassandra() {
     @Test
     public void cassandraMetricsArePublished() {
         assertThat(app.getEnvironment().metrics().getNames(), hasItem("s1.continuous-cql-requests"));
-        assertThat(app.getEnvironment().metrics().getNames(), hasItem("s1.nodes.localhost:" +
-                EmbeddedCassandraServerHelper.getNativeTransportPort() +".bytes-sent"));
+        assertThat(app.getEnvironment().metrics().getNames(), hasItem("bytes-sent"));
```

Comment:
I don't think `hasItem()` will work, because it checks equality, not whether one of the items in the array **contains** that.